sdk/tests/tutorials/ibor/test_cut_labels.py

```diff
@@ -0,0 +1,231 @@
+import unittest
+from datetime import datetime, date, timedelta
+
+import pytz
+import uuid
+
+import lusid
+import lusid.models as models
+
+from lusid.utilities.api_client_builder import ApiClientBuilder
+from utilities.instrument_loader import InstrumentLoader
+from utilities.test_data_utilities import TestDataUtilities
+from utilities.credentials_source import CredentialsSource
+
+
+class CutLabels(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.scopes_api = lusid.ScopesApi(api_client)
```

Comment:
You don't use this scopes_api, so you can remove it.

Reply:
Done